tests/real-examples/real-examples.yaml

```diff
@@ -27,6 +27,10 @@ links:
     comment: ignore 'npm install -d' & find '/usr/local/bin/npm install jade'
     post: answer
     registry: npm
+  https://stackoverflow.com/questions/51508364:
+    comment: ignore 'yarn add <yourPackage> --network-timeout 100000'
+    post: answer
+    registry: npm
```

Comment:
This file is an input for fetching real StackOverflow pages and save the found results in `tests/real-examples/real-examples-results.yaml`. It's mean that if you changed this file, I expect the "results" file to be changed too.

To run the test and compare the results you need to run `rm -r tests/real-examples/snapshots && yarn test:integration` to regenerate the snapshots and run the test.

Then you will see the changes in the "results" file (with Git)